### ISSUE TITLE
Parallax must be easy to turn off.

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/hourglass#249]
## Report Issue
**Describe the inaccessible feature**
Parallax is being used for non-essential movement.   Non-essential animation must be easy to turn off.  It is not enough to expect any impacted users to have turned such off within their own browser.

## Preferred Solution

either: users should opt in to parallax, with the default being that it is off.

Or: 
There should be multiple places where users can opt out of parallax, and if it has been turned off this setting should be remembered for that user where that fits with cookie policies etc.  

- at top of any page where parallax is in use - such that it can be halted before being encountered.
- on the home page, regardless of whether in use, as this is the most likely landing page for new users.
- alongside wherever parallax is in use, such that it can be stopped when first encountered. 

## Additional context
Parallax effect is considered an animation.  [see discussion here of WCAG 2.2.2. Pause, Stop, Hide ](https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html)
